<ui version="4.0" >
 <class>FiletypeAssociationDialog</class>
 <widget class="QDialog" name="FiletypeAssociationDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Filetype Associations</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="_2" >
   <item>
    <widget class="QTreeWidget" name="filetypeAssociationList" >
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <property name="itemsExpandable" >
      <bool>false</bool>
     </property>
     <property name="sortingEnabled" >
      <bool>true</bool>
     </property>
     <column>
      <property name="text" >
       <string>Filename Pattern</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Filetype</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string/>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="_3" >
     <item row="1" column="2" >
      <widget class="QPushButton" name="deleteAssociationButton" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="toolTip" >
        <string>Press to delete the selected association</string>
       </property>
       <property name="text" >
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QComboBox" name="filetypeCombo" >
       <property name="toolTip" >
        <string>Select the filetype to associate</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="textLabel3_3" >
       <property name="text" >
        <string>Filetype:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="filePatternEdit" >
       <property name="toolTip" >
        <string>Enter the filename pattern to be associated</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="textLabel2_6" >
       <property name="text" >
        <string>Filename Pattern:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <widget class="QPushButton" name="addAssociationButton" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="toolTip" >
        <string>Press to add or change the entered association</string>
       </property>
       <property name="text" >
        <string>Add/Change</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="6" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>filetypeAssociationList</tabstop>
  <tabstop>filePatternEdit</tabstop>
  <tabstop>filetypeCombo</tabstop>
  <tabstop>addAssociationButton</tabstop>
  <tabstop>deleteAssociationButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FiletypeAssociationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>84</x>
     <y>554</y>
    </hint>
    <hint type="destinationlabel" >
     <x>88</x>
     <y>575</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FiletypeAssociationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>213</x>
     <y>553</y>
    </hint>
    <hint type="destinationlabel" >
     <x>215</x>
     <y>575</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
